### PR TITLE
Cron piece

### PR DESCRIPTION
**Pull request by joshualyon7**
Ok so big boy pull request, but the majority of the files changed was updating `@lib/util` imports to be `@lib/util/generalUtils` imports

### CODE REVIEW

File: src/commands/fun/poll.ts [hunk 1]
```diff
@@ -54,9 +57,12 @@ export default class extends Command {
 		const timespan = parse(interaction.options.getString('timespan'));
 		const question = interaction.options.getString('question');
 		const choices = interaction.options.getString('choices').split('|').map(choice => choice.trim());
+		if (!args.includes(interaction.options.getString('optiontype'))) {
```
Comment (sbrugel) [inline src/commands/fun/poll.ts:60]: This is a required field, so why would this be necessary? Just something to satisfy typescript/eslint?

File: src/commands/fun/poll.ts [hunk 2]
```diff
@@ -182,3 +144,100 @@ export default class extends Command {
...
+	if (prevAnswers.length === 0 || !prevAnswers.includes(newChoice)) {
+		await i.reply({ ephemeral: true, content: `Vote for ***${newChoice}*** recorded. To remove it, click the same option again.` });
+	}
+	if (!i.replied) await i.reply({ ephemeral: true, content: `Vote for ${newChoice} removed.` });
+
+	await i.client.mongo.collection<Poll>(DB.POLLS).findOneAndReplace({ message: newPoll.message }, newPoll);
+	return;
+}
+
+/**
```
Comment (sbrugel) [inline src/commands/fun/poll.ts:212]: very noice doc comments

**Review by sbrugel — COMMENTED** [review]
- What happened to src/pieces/remind.ts? Or is this just a redundant file?
- Otherwise looks good. I'm going to assume the utils reference change is just a minor restructuring of things for the new interaction stuff coming in

**Comment by joshualyon7** [comment]
> * What happened to src/pieces/remind.ts? Or is this just a redundant file?
> 
>     * Otherwise looks good. I'm going to assume the utils reference change is just a minor restructuring of things for the new interaction stuff coming in

Got moved into `src/pieces/tasks.ts`

File: src/commands/fun/poll.ts (hunk 1 above)
Comment (joshualyon7) [reply src/commands/fun/poll.ts:60]: This is more for a developer touching it. I do stuff like [this](https://github.com/ud-cis-discord/SageV2/pull/266/files#diff-71480bc3b9d8496b951d2aa274bcda13f86db223ae34c94015a2f5bb3d226f7dR63) a bunch so it's just making sure when I check the option type against `"Single" | "Multiple"` it *actually is* one of those types

Comment (joshualyon7) [reply src/commands/fun/poll.ts:60]: To be clear, there's probably definitely a better way to constrain stuff later but djs doesn't let us set union types as argument choices yet :(

Comment (sbrugel) [reply src/commands/fun/poll.ts:60]: Ok yeah, I see now

**Review by renross — APPROVED** [review]

**Review by sbrugel — APPROVED** [review]

**Review by sbrugel — APPROVED** [review]

**Review by renross — APPROVED** [review]

**Pull request merged**